src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,74 @@
+package duke;
+import java.util.ArrayList;
```

Comment:
It would be good if there was a line spacing between the package and the imports.

Reply:
I've fixed this in the latest commit as well, thanks for letting me know!